x/dualstaking/keeper/delegate.go

```diff
@@ -214,18 +205,19 @@ func (k Keeper) modifyStakeEntryDelegation(ctx sdk.Context, delegator, provider,
 	}
 
 	details := map[string]string{
-		"provider":        stakeEntry.Address,
-		"chain_id":        stakeEntry.Chain,
-		"moniker":         stakeEntry.Moniker,
-		"stake":           stakeEntry.Stake.String(),
-		"effective_stake": stakeEntry.EffectiveStake().String() + stakeEntry.Stake.Denom,
+		"provider_vault":    stakeEntry.Vault,
+		"provider_operator": stakeEntry.Operator,
+		"chain_id":          stakeEntry.Chain,
+		"moniker":           stakeEntry.Moniker,
+		"stake":             stakeEntry.Stake.String(),
+		"effective_stake":   stakeEntry.EffectiveStake().String() + stakeEntry.Stake.Denom,
 	}
 
 	if stakeEntry.Stake.IsLT(k.GetParams(ctx).MinSelfDelegation) {
-		err = k.epochstorageKeeper.RemoveStakeEntryCurrent(ctx, chainID, stakeEntry.Address)
+		err = k.epochstorageKeeper.RemoveStakeEntryCurrent(ctx, chainID, stakeEntry.Operator)
 		if err != nil {
 			return utils.LavaFormatError("can't remove stake Entry after decreasing provider self delegation", err,
-				utils.Attribute{Key: "provider", Value: stakeEntry.Address},
+				utils.Attribute{Key: "provider", Value: stakeEntry.Operator},
 				utils.Attribute{Key: "spec", Value: chainID},
```

Comment:
I agree that in the RemoveStakeEntryCurrent call I should use vault, but in the error I'll keep the operator address because this is the address that the user uses. see https://github.com/lavanet/lava/commit/17fdee4530cf033ed460f028077d7f0442033d57